org.eclipse.dawnsci.analysis.examples/src/org/eclipse/dawnsci/analysis/examples/io/LoadingExamples.java

```diff
@@ -39,7 +39,7 @@
 	private static ILoaderService service;
 	
 	// Injected by OSGi
-	public static void setLoaderService(ILoaderService s) {
+	public void setLoaderService(ILoaderService s) {
 		service = s;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)